### ISSUE TITLE
help me pls

### ISSUE THREAD

**Issue by Averox402** [dalaranwow/dalaran-wow#5961]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. I bought it 30 min ago a dagger from the auction house and it didn't appear in my bag
2. 1 hour ago I was in the progress of doing the ''Klaven's tower'' mission and i didn't get the key for the chest there so I couldn't complete the quest
3. i am new at wow so idk the names of the mobs. etc.
![World of Warcraft 13_06_2021 20_08_35](https://user-images.githubusercontent.com/85841921/121816240-2b0d3500-cc83-11eb-8162-4151abd65c9a.png)

![World of Warcraft 13_06_2021 20_08_35](https://user-images.githubusercontent.com/85841921/121816240-2b0d3500-cc83-11eb-8162-4151abd65c9a.png)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Hello @Averox402,

Regarding your issue with the auction house, please make sure to check your mail inbox, the item you bought should be there. If the issue persists, please make an in-game ticket explaining your issue with details and we will check it out.

As for the Klaven's Tower quest, it works as intended. The chest isn't meant to be opened with the key. Read internet sources such as Wowhead if you don't know how to proceed and I'm sure you will be able to complete it.

Cheers, closing up.

**Comment by Averox402** [comment]
(Tnx for the respond, I really appreciate that.) Idk if it's bug or my pc
problem but I logged in with my account and it didn't logged me in.

בתאריך יום א׳, 13 ביוני 2021, 23:40, מאת Golganeth ‏<
***@***.***>:
